### PR TITLE
Adjusted versions

### PR DESCRIPTION
**Pull request by Yuri05**
Text correction in the landscape picture (wrongly placed brackets)

### CODE REVIEW

File: OSP_software_landscape.svg
Comment (Yuri05) [inline OSP_software_landscape.svg:1]: <details><summary>The change is here 🙂 </summary>

grafik

</details>

**Comment by msevestre** [comment]
The build failed for some reason??

Kind Regards,

Michael


--


*Michael Sevestre*

*Design2Code Inc.*


Email ***@***.***

Website www.design2code.ca

**Comment by Yuri05** [comment]
no, somehow it cannot download the setups? probably a temporary AppVeyour issue

**Comment by Yuri05** [comment]
worked after build restart

**Pull request merged**